control-plane-operator/controllers/hostedcontrolplane/v2/kas/config.go

```diff
@@ -227,6 +227,7 @@ func generateConfig(p KubeAPIServerConfigParams) *kcpv1.KubeAPIServerConfig {
 	args.Set("service-account-signing-key-file", cpath(serviceAccountKeyVolumeName, pki.ServiceSignerPrivateKey))
 	args.Set("service-node-port-range", p.NodePortRange)
 	args.Set("shutdown-delay-duration", "70s")
+	args.Set("shutdown-watch-termination-grace-period", "25s")
```

Comment:
Based on the unit test failure, I suspect that you need to update the config hash in the test fixtures.